WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs

```diff
@@ -390,14 +390,14 @@ public async Task LoadWalletAsync()
 					// Successfully initialized.
 					Owner.OnClose();
 				}
+				catch (OperationCanceledException ex)
+				{
+					Logger.LogTrace(ex);
```

Comment:
Just a question: don't we need `NotificationHelpers.Error(ex.ToUserFriendlyString());` above this?

Reply:
The exception is normal because of OperationCanceledException means that the task has been canceled intentionally.